src/common/services/request/index.js

```diff
@@ -4,7 +4,7 @@ import {
 } from "json-graphql-parser/v2/index.js";
 
 const BACKEND_URL = `${process.env.REACT_APP_NHOST_BACKEND_URL}/${process.env.REACT_APP_NHOST_VERSION}/${process.env.REACT_APP_NHOST_ENDPOINT}`;
-const header = {
+export const header = {
```

Comment:
Why export is required here?

Reply:
@koustov, I was not sure about this and it's not being used anywhere that's why I added export.